packages/react-components/src/layout/src/Stack.tsx

```diff
@@ -52,12 +57,12 @@ export function InnerStack({
     children,
     forwardedRef,
     ...rest
-}) {
+}: InnerStackProps): ReactElement {
     const alignProps = useFlexAlignment("vertical", align, verticalAlign);
 
     return (
         <Flex
-            {...mergeProps(
+            {...mergeProps<Partial<FlexProps>[]>(
```

Comment:
Do you really have to specify a type to mergeProps? Things will get ugly real quick if we must do this everywhere.